dataactcore/scripts/setupValidationDB.py

```diff
@@ -1,47 +1,34 @@
+from dataactbroker.app import createApp
+from dataactcore.interfaces.db import GlobalDB
+from dataactcore.models import lookups
 from dataactcore.models.validationModels import FileTypeValidation, FieldType, RuleSeverity
-from dataactcore.interfaces.db import databaseSession
 
 
 def setupValidationDB():
     """Create validation tables from model metadata and do initial inserts."""
-    with databaseSession() as sess:
+    with createApp().app_context():
+        sess = GlobalDB.db().session
         insertCodes(sess)
         sess.commit()
 
+
 def insertCodes(sess):
     """Insert static data."""
-
-    # insert file types
-    fileTypeList = [
-        (1, 'appropriations', 'appropriations file',1),
-        (2, 'program_activity','program activity and object class file',2),
-        (3, 'award_financial', 'award_financial file',3),
-        (4, 'award', 'award file',4),
-        (5, 'award_procurement', 'award procurement file', 5)
-        ]
-    for f in fileTypeList:
-        fileType = FileTypeValidation(file_id=f[0], name=f[1], description=f[2], file_order = f[3])
-        sess.merge(fileType)
+    # insert validation file type
+    # todo: combine this table with file_types table
+    for f in lookups.FILE_TYPE:
+        if f.order:
```

Comment:
E and F aren't in the current `file_type_validation` table, so I didn't load them here. Not fully understanding the ramifications of messing with the two file type tables, I tried to leave them as is